### PR TITLE
Unify docs

### PR DESCRIPTION
**Pull request by DavidS**
* The PROXY.md file is not shown on the forge
* The `doc/` directory is used by puppet-strings
* The information in the file is outdated as the bug in rubygems has been fixed a long time ago

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,9 +147,34 @@ _Note: It is recommended you migrate to using the `pe_r10k` module which is basi
 a clone of this modules features and file tickets for anything missing._
 
 
-## Usage
+### Using a internal gem server
```
Comment (igalic) [inline README.md]: *an

File: README.md [hunk 2]
```diff
@@ -147,9 +147,34 @@ _Note: It is recommended you migrate to using the `pe_r10k` module which is basi
...
 
-Installing using a proxy server
+```puppet
+# Create a global gemrc for Puppet Enterprise to add the local gem source
+# See http://projects.puppetlabs.com/issues/18053#note-12 for more information.
+
```
Comment (igalic) [inline README.md]: why not leave this as text? rather than code

File: README.md (hunk 1 above)
Comment (DavidS) [reply README.md]: :+1: fixed

File: README.md (hunk 2 above)
Comment (DavidS) [reply README.md]: why? copy&paste.

fixed it anyways.

**Comment by bastelfreak** [comment]
thanks!

**Pull request merged**